### PR TITLE
Cms content entry - ship updated data api intro and basic FAQ

### PR DESCRIPTION
**Pull request by ghop02**

### CODE REVIEW

File: src/cms-content/learn/products-landing.json [hunk 1]
```diff
@@ -1,11 +1,11 @@
 {
   "header": "Data API",
-  "intro": "Looking for all the data that powers our site in a machine-readable format? You can use our API, made for ease-of-use by those wishing to programmatically ingest our data.",
+  "intro": "The Covid Act Now API provides access to comprehensive COVID data - both current and historical. The data is available for all US states, counties, and metros and is aggregated from a number of official sources, quality assured, and updated daily. It is available in JSON and CSV formats.",
```
Comment (mikelehen) [inline src/cms-content/learn/products-landing.json:3]: FWIW, I think `data&mdash;both` would be slightly more correct hyphen / dash usage.

Also, I think `quality-assured` should probably be hyphenated?

File: src/cms-content/learn/products-landing.json [hunk 2]
```diff
@@ -55,6 +55,12 @@
         }
       ]
     },
+    {
+      "productName": "FAQs",
```
Comment (mikelehen) [inline src/cms-content/learn/products-landing.json:59]: Oh boy, we are abusing this schema a bit. 🤣

File: src/cms-content/learn/products-landing.json [hunk 3]
```diff
@@ -55,6 +55,12 @@
...
+      "productId": "faqs",
+      "productSubtitle": "Frequently Asked Questions",
+      "productDescription": "### Where do you source your data from?\n\nOur data comes from official sources including the U.S. Department of Health and Human Services, the Centers for Disease Control and Prevention, The New York Times, and official state and county dashboards.\n\n### Where can I see all data fields included in your API?\n\nAll fields can be found in our [data definitions](https://apidocs.covidactnow.org/data-definitions).  To see an overview of how to query data for states, counties, or metros visit our [getting started guide](https://apidocs.covidactnow.org/access#getting-started).\n\n### When does your data update and at what time?\n\nOur data is updated daily around Noon Eastern time.  Updates reflects the latest available data from each our our sources."
```
Comment (mikelehen) [inline src/cms-content/learn/products-landing.json:62]: typo: "each our our sources" => "each of our sources"

How weird is it to throw "New York Times" in the middle of a list of "official sources"? Maybe fine. 🤷

**Review by mikelehen — APPROVED** [review]

File: src/cms-content/learn/products-landing.json (hunk 2 above)
Comment (ghop02) [reply src/cms-content/learn/products-landing.json:59]: heh its gone through a couple revisions at this point

Comment (ghop02) [reply src/cms-content/learn/products-landing.json:59]: but on my list of things to do

File: src/cms-content/learn/products-landing.json (hunk 3 above)
Comment (ghop02) [reply src/cms-content/learn/products-landing.json:62]: heh I think we use this text elsewhere

**Pull request merged**